pkg/networkutils/network.go

```diff
@@ -929,3 +935,27 @@ func (n *linuxNetwork) UpdateRuleListBySrc(ruleList []netlink.Rule, src net.IPNe
 	}
 	return nil
 }
+
+// GetEthernetMTU gets the MTU setting from AWS_VPC_ENI_MTU, or defaults to 9001 if not set.
+func GetEthernetMTU() int {
+	if envMTUValue := os.Getenv(envMTU); envMTUValue != "" {
+		mtu, err := strconv.Atoi(envMTUValue)
+		if err != nil {
+			log.Error("Failed to parse "+envMTU+"; will use ", maximumMTU, err.Error())
+			return maximumMTU
+		}
+		// Restrict range between jumbo frame and the maximum required size to assemble.
+		// Details in https://tools.ietf.org/html/rfc879 and
+		// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/network_mtu.html
+		if mtu < minimumMTU {
+			log.Error(""+envMTU+" too low; will use ", minimumMTU)
```

Comment:
Why are you appending empty string in front of envMTU?

Reply:
That was just sloppy, copied from the previous error log line. I've updated to use `log.Errorf()`